src/elements/List/List.js

```diff
@@ -1,32 +1,44 @@
-import React, { Component, PropTypes } from 'react'
-import classNames from 'classnames'
+import cx from 'classnames'
+import React, { PropTypes } from 'react'
 import META from '../../utils/Meta'
+import {getUnhandledProps} from '../../utils/propUtils'
 import ListItem from './ListItem'
 
-export default class List extends Component {
-  static propTypes = {
-    children: PropTypes.node,
-    className: PropTypes.string,
-  }
-
-  static _meta = {
-    library: META.library.semanticUI,
-    name: 'List',
-    type: META.type.element,
-  }
-
-  static Item = ListItem
-
-  render() {
-    const classes = classNames(
-      'ui',
-      this.props.className,
-      'list'
-    )
-    return (
-      <div {...this.props} className={classes}>
-        {this.props.children}
-      </div>
-    )
-  }
+function _List(props) {
```

Comment:
Indeed.  It is a base header component to be reused among the various header components. 

>On a side note, the header subcomponents do not really follow our API pattern and should be rethought.